api/src/events.rs

```diff
@@ -186,8 +202,8 @@ pub async fn process_drop_minted_event(
 
     let mut purchase_am: purchases::ActiveModel = purchase.into();
 
-    purchase_am.status = Set(tx_status.try_into()?);
-    purchase_am.tx_signature = Set(Some(tx_signature));
+    purchase_am.status = Set(status);
+    purchase_am.tx_signature = Set(signature);
     purchase_am.update(db.get()).await?;
 
```

Comment:
NB: we need to rename purchases to something like mints and collection mints to editions where mint represent the initial minting of the nft and edition is the edition.